package.json

```diff
@@ -96,7 +96,7 @@
     "eslint-config-davidtheclark-node": "^0.2.2",
     "eslint-config-prettier": "^2.9.0",
     "eslint-plugin-flowtype": "^2.39.1",
-    "eslint-plugin-node": "^5.2.1",
+    "eslint-plugin-node": "9.1.0",
```

Comment:
Any reason not to use a caret here, like the other `devDependencies`?

Reply:
Nope, just a mistake. I rebased to master and this isn't needed anymore